### DISCUSSION TITLE
Nextcloud Snap

### DISCUSSION THREAD

**Discussion by martinjr85** [nextcloud/recognize#104] [General]
Has anyone got this to work successfully with the Nextcloud Snap installation? I'm seeing lots of permission denied errors in the logs.

**Comment by marcelklehr** [comment]
Mmmh, I generally don't recommend using the snaps for server software as it's hard to figure out what's going on in case of problems

**Comment by bigrjsuto** [comment]
I would also like to know if anyone has gotten this to work on a snap installation.
